Library/Formula/ffts.rb

```diff
@@ -2,8 +2,9 @@ class Ffts < Formula
   desc "C library that computes the discrete Fourier transform"
   homepage "http://anthonix.com/ffts/"
   head "https://github.com/anthonix/ffts.git"
-  url "http://anthonix.com/ffts/releases/ffts-0.7.tar.gz"
-  sha256 "6362e498e5aa241661cfe9060153076b78e300a5d3c365997ba6ac2f637df3ff"
+  url "https://github.com/anthonix/ffts/archive/fa1780c68593762b1e4bdbc46d83912db3eba27a.tar.gz"
```

Comment:
The homepage is also dead & there hasn't been an upstream commit in 18+ months, so this may end up as a bit of a rabbit-hole.

We could move it to `homebrew/homebrew-head-only` and remove the release for now.